season/conditionals.md

```diff
@@ -8,15 +8,20 @@ Chartreuse is my favorite color. Magenta is my second favorite. Periwinkle is my
 ```ruby
 color = "Magenta"
 
-# write your code here
+if color == "Magenta"
+	puts "#{color} is my favorite color."
+end 
 ```
 
 Now, turn your color conditional into a method called `color_checker` that takes any color as an argument.
 
 ```ruby
 def color_checker(color)
-  # write your code here
+   "#{color}"
 end
+
+puts color_checker("BLACK")
+
```

Comment:
So, this is just going to return the color itself. My directions were unclear, but I meant for you to make a method that took a color and applied the same logic as the conditional above (i.e. wrap your conditional in a method).